plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -126,5 +127,19 @@ func (a *quotaAdmission) Admit(attr admission.Attributes) (err error) {
 	if attr.GetSubresource() != "" {
 		return nil
 	}
+	createUninitialized, err := util.IsCreatingUninitialized(attr)
+	if err != nil {
+		return err
+	}
+	initialized, err := util.IsObjectInitialized(attr.GetObject())
+	if err != nil {
+		return err
+	}
+	// Object count quota should be charged when the object is created, no
+	// matter if the object is initialized or not.
+	// Other quotas should be charged after the object is initialized
```

Comment:
this code doesn't seem to match this comment

Reply:
it seems like we should continue to call the evaluator, not assume initialization is the only aspect being considered by it and short-circuit here

Reply:
The plugin used to care about CREATE and UPDATE. Now I make it ignore the intermediate updates during initialization. I think it's safe.

The code will just work if I remove the short-circuit here, but that causes many unnecessary calls to the evaluators. Can we keep the short-circuit until we have a use case?